dex/meter/meterer_test.go

```diff
@@ -43,22 +39,24 @@ func TestDelayedRelay(t *testing.T) {
 		t.Fatalf("wrong signal count. expected 1, got %d", n)
 	}
 
-	// Sending two signals simultaneously should end up with two signals, at
-	// least minDelay apart.
+	// 2nd signal should be delayed at least for minDelay.
+	sentTime := time.Now()
```

Comment:
This *is* the root of the issue with the test. The signal receiver can receive as lazily as it likes.